src/Cli/dotnet/CommonOptions.cs

```diff
@@ -1,16 +1,16 @@
 // Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
+using Microsoft.DotNet.Cli.Utils;
```

Comment:
Minor nit: Most of these repos sort usings where System namespaces appear first. There's a VS setting / .editorconfig that will do that for you. I'm surprised the .editorconfig didn't kick in: https://github.com/dotnet/sdk/blob/main/.editorconfig#L81

Reply:
would dotnet-format apply these as well? if so it's probably a good practice for me to start running that over changed files before commit.